File: app/src/main/kotlin/com/moviereel/data/api/model/Networks.java
package com.moviereel.data.api.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author lusinabrian on 10/04/17
 */

public class Networks {
    @Expose
    @SerializedName("id")
    private int id;

    @Expose
    @SerializedName("name")
    private String name;
}

File: remote/src/main/kotlin/com/moviereel/remote/models/Networks.kt
package com.moviereel.remote.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author lusinabrian on 10/04/17
 */

data class Networks(
        @Expose
        @SerializedName("id")
        private val id: Int = 0,

        @Expose
        @SerializedName("name")
        private val name: String? = null
)
